types/stylelint/index.d.ts

```diff
@@ -741,6 +745,17 @@ declare namespace stylelint {
 		source: string;
 	};
 
+	export type EditInfo = {
+		/**
+		 * The start and end offset of the edit. This is the range that is expected to be sliced out.
```

Comment:
Can you add that the pair are "0-based indices"?
cf https://eslint.org/docs/latest/integrate/nodejs-api#-editinfo-type

Reply:
I've replaced it fully with the ESLint wording.